ldms/python/ldms.pxd

```diff
@@ -486,6 +494,13 @@ cdef extern from "ldms.h" nogil:
     int ldms_schema_meta_array_add(ldms_schema_t s, const char *name,
                                    ldms_value_type t, uint32_t count)
 
+    # --- schema hash / digest --- #
+    ctypedef unsigned char *ldms_digest_t
+    ldms_digest_t ldms_set_digest_get(ldms_set_t s)
```

Comment:
Are we missing ``` int ldms_set_digest_cmp(ldms_digest_t a, ldms_digest_t b)```

Reply:
I'll add it.

Reply:
Correction. We already have `int ldms_digest_cmp(ldms_digest_t a, ldms_digest_t b)`